sources/3.0.0/sections/12-metadata.adoc

```diff
@@ -511,12 +510,16 @@ CI_Responsibility>CI_Organisation
 |Information about the frequency and scope of resource updates
 |0..1
 |
-|S-100 restricts the multiplicity to 0..1 and adds specific restrictions on the ISO 19115 structure and content. See <iho-s100,part=17>>. +
+|S-100 restricts the multiplicity to 0..1 and adds specific restrictions on the ISO 19115 structure and content. See <<iho-s100,part=17>>. +
 Format: PnYnMnDTnHnMnS (XML built-in type for ISO 8601 duration). See <<iho-s100,part=17,clause=17-4.9>>. +
 *S-102 discovery metadata blocks should populate maintenance information if and only if the date of the next edition is definite, whether it is due on a regular or irregular schedule.*
 
 |===
-
+[[subsec-s102-replacedData-dataReplacement-note1]]
```

Comment:
Please stop copying everything from the S-100. This only makes the maintenance of the document more difficult because every word has to be checked. A reference to the relevant passage in the S-100 should be sufficient in the comment.

Reply:
Agreed. Removed quote and changed references to point directly to the same language in S-100.